index.html

```diff
@@ -1512,113 +1499,146 @@ <h1>WoT Architecture</h1>
+        </p>
         <section id="sec-architecture-overview">
             <h2>Overview</h2>
-            <p>A WoT system conceptually consists of things that
-                communicate with each other over network connections.
-                These things can take different logical roles in the
-                communication:</p>
-            <ul>
-                <li><a>WoT Client</a>: A thing can behave as a
-                    client, which is an active component and issues
-                    requests to other things. Requests use the
-                    functionalities that are provided by the other
-                    things, we call this interaction <em>to consume
-                        a thing</em>.</li>
-                <li><a>WoT Server</a>: A thing can behave as a
-                    server, which is a component that reacts to requests
-                    from other things. Requests query or modify the
-                    state of a thing, we call this interaction <em>to
-                        expose a thing</em>. Note that the server, is not
-                    purely a passive reactive component, since it can
-                    offer a subscribe/notify mechanism, which allows
-                    clients to be notified about state changes of a
-                    thing.</li>
-                <li><a>WoT Servient</a>: A thing can behave both
-                    as a client and a server, i.e. it can <em>expose</em>
-                    functionality to other things and it can <em>consume</em>
-                    functionalities that are provided by other things.</li>
-            </ul>
-
-            <p>Note that these concepts are logical definitions
-                based on the tasks of a thing. They do not require that
-                an exposed thing must be implemented with a (Web) server
-                and a consumed thing as a (Web) client.</p>
+                A <a>Thing</a> is the abstraction of a physical or virtual entity (e.g., a device or a room) and is described by standardized metadata.
+                In W3C WoT, the description metadata MUST be a WoT Thing Description (TD) [[!wot-thing-description]].
+                <a>Consumers</a> MUST be able to process the <a>WoT Thing Description</a> format, which is based on JSON [[!RFC8259]].
+                The format can be processed either through classic JSON libraries or a JSON-LD processor, as the underlying information model is graph-based and its serialization compatible with JSON-LD 1.1 [[?json-ld-syntax]].
+                A <a>TD</a> is instance-specific (i.e., describes an individual Thing, not types of Things)
+                and is the default Web representation of a <a>Thing</a>.
+                There MAY be other Web representations of a <a>Thing</a> such as an HTML-based user interface or simply an image of the physical entity.
```

Comment:
Please consider to generalize the term "Web representation" - it is not Web-specific and could be used for closed (non-Web) scenarios as well.
You could call it external representation.

Reply:
On the other hand, such representations with Media Type are exactly one of those "Web technologies" which found application in the other systems you are hinting at.

Reply:
Now added or even non-Web representations in closed systems